### ISSUE TITLE
TextSpans with gesture detectors are not accessible

### ISSUE THREAD

**Issue by alanrussian** [flutter/flutter#20643]
Let's say you have some text with a link inside. You can create it like so:

```dart
RichText(
  text: TextSpan(
    children: [
      TextSpan(text: 'blah'),
      TextSpan(
        text: ' ${ExpressStrings.learnMore}',
        recognizer: TapGestureRecognizer()
          ..onTap = () {
            // Action...
          },
      ),
    ],
  ),
);
```

There are two accessibility issues here:

- It's currently not possible to focus on the text spans with the gesture recognizer. 

- Assuming a smaller font size, the touch targets for these gesture recognizers don't meet the minimum touch target size (48x48). I would expect that this be fixed by making an invisible touch target area larger than the visual one (i.e., no visual change).

/cc @goderbauer

**Comment by jonahwilliams** [comment]
* When we are building the TextSpans we are currently just dumping all of the strings into a single node - instead we need to check if there are recognizers and create additional nodes. This should work with no further modifications to the accessibility system.
* In general it is not possible to meet the 48 by 48 guidelines with links. The current guidelines have a specific carve-out for links.
